app/models/shared_link.rb

```diff
@@ -0,0 +1,11 @@
+class SharedLink < ApplicationRecord
+  belongs_to :user
+
+  scope :available, -> {
+    where("expired_at > ?", Time.zone.now)
```

Comment:
口頭で話して Sokuseki のコードベースでは `Time.current` で統一しましょうか、となった :memo:

```suggestion
    where("expired_at > ?", Time.current)
```